### PR TITLE
Add deployment part on gunicorn

### PR DESCRIPTION
**Pull request by rmoorman**
This adds a short explanation on how to use gunicorn together with
a pyramid based application.

### CODE REVIEW

File: docs/deployment/gunicorn.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
+********
+gunicorn
+********
+
+The short story
+===============
+Running your pyramid based application with gunicorn can be as easy as::
+
+    gunicorn --paste production.ini
+
```
Comment (stevepiercy) [inline docs/deployment/gunicorn.rst]: Are there any settings in the `ini` that are specific to gunicorn? If so, it would be good to include examples here.

Comment (rmoorman) [inline docs/deployment/gunicorn.rst]: IIRC it is indeed possible to add gunicorn settings to the `ini` file too. I will check and add it to the information on the page and let you know :-)

**Comment by rmoorman** [comment]
@stevepiercy I just added some information about gunicorn specific settings that can be included in the INI file.

**Comment by stevepiercy** [comment]
Thanks @rmoorman!

**Pull request merged**